build.gradle

```diff
@@ -89,6 +89,7 @@ task installGitHooks(type: Copy, dependsOn: 'setLocalRepo') {
 
 allprojects {
     apply plugin: 'java'
+    apply plugin: 'maven-publish'
```

Comment:
We probably only want to apply the plugin to the `checker` module, but it needs to be applied before the shadow plugin.  So we need to add some kind of conditional here.

Reply:
Or actually, maybe we want this to apply to all the modules?  Not sure.  If we want it everywhere, we'll have to move and/or copy the pom configuration from `build/checker.gradle`